app/src/views/campaigns/settings.vue

```diff
@@ -1,10 +1,217 @@
+              { validator: dateValidator, trigger: 'blur' },
+            ]"
+          >
+            <el-date-picker
+              v-model="tempData.endAt"
+              placeholder="Date de fin"
+              type="date"
+            />
+          </el-form-item>
+        </el-col>
+      </el-row>
+      <el-row :gutter="20">
+        <el-col :span="12">
+          <el-form-item
+            :label="$t('table.campaigns.maxProposition')"
+            prop="maxProposition"
+          >
+            <el-input-number
+              :min="0"
+              :max="50"
+              v-model="tempData.maxProposition"
+            />
+          </el-form-item>
+        </el-col>
+      </el-row>
+
+      <div class="dialog-footer">
+        <el-button type="danger" @click="archive">{{ $t('table.archive') }}</el-button>
+        <el-button @click="reset">{{ $t('table.reset') }}</el-button>
+        <el-button type="primary" @click="updateData">{{ $t('table.confirm') }}</el-button>
+      </div>
+    </el-form>
 
+    <progress-dialog ref="ProgressDialog" @close-after-success="reset" />
+  </div>
+</template>
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
 
-@Component({ name: 'CampaignsSettings' })
-export default class extends Vue {}
+import moment from 'moment';
+import { Component, Vue, Watch } from 'vue-property-decorator';
+import { cloneDeep } from 'lodash';
+import { Form } from 'element-ui';
+
+import { defaultCampaignData, createCampaign, deleteCampaign, getCampaign, updateCampaign } from '../../api/campaigns';
+import CategorySelect from '@/components/CategorySelect/index.vue';
+import ProgressDialog from '@/components/ProgressDialog/index.vue';
+import { CampaignsModule } from '../../store/modules/campaigns';
+
+
+@Component({ 
+  name: 'CampaignsSettings' ,
+  components: { ProgressDialog, CategorySelect },
+})
+
+export default class extends Vue {
+  private tempData: any = defaultCampaignData; 
+  
+  private cnt: number = 0;
+  private max: number = 1;
+ 
+  public get id() {
+    return Number(this.$route.params.id);
+  }
+
+  public created() {
+    this.$nextTick(() => {
+      this.setupCampaign();
+    });
+  }
+
+  @Watch('id')
+  public async setupCampaign() {
+    let tmp: any;
+    if(this.id) {
+      tmp = await CampaignsModule.getCampaign(this.id);
+    }
+    tmp = cloneDeep(tmp || defaultCampaignData);
+
+    this.tempData = tmp;
+  }
+
+  private updateData() {
+     (this.$refs['dataForm'] as Form).validate(async valid => {
+       if (valid) {
+         const temp = Object.assign({}, this.tempData);
+
+         temp.endAt= moment(temp.endAt).valueOf();
+         delete temp.startAt;
+
+         await updateCampaign(temp.id!, temp);
+        this.$notify({
+          title: this.$t('notify.campaigns.update.title') as string,
+          message: this.$t('notify.campaigns.update.msg') as string,
+          type: 'success',
```

Comment:
These notifications translations aren't created

Reply:
I guess I forgot to push the translation file. My bad. Please, let me know if it works with the second push.

Reply:
Nop, you just forget to put translation in this file